test/adapter/json_api/links_test.rb

```diff
@@ -7,18 +7,24 @@ class LinksTest < ActiveSupport::TestCase
         LinkAuthor = Class.new(::Model)
         class LinkAuthorSerializer < ActiveModel::Serializer
           link :self do
-            href "//example.com/link_author/#{object.id}"
+            href "http://example.com/link_author/#{object.id}"
```

Comment:
It's been a point of discussion. https://github.com/json-api/json-api/issues/898